packages/esm-appointments-app/src/home-appointments/appointments-list.component.tsx

```diff
@@ -212,50 +263,115 @@ const AppointmentsBaseTable = () => {
   }
 
   return (
-    <div className={styles.homeAppointmentsContainer}>
-      <div className={styles.headerContainer}>
-        <div className={!isDesktop(layout) ? styles.tabletHeading : styles.desktopHeading}>
-          <h4>{t('todaysAppointments', "Today's Appointments")}</h4>
+    <>
+      <div className={styles.homeAppointmentsContainer}>
+        <div className={styles.headerContainer}>
+          <div className={!isDesktop(layout) ? styles.tabletHeading : styles.desktopHeading}>
```

Comment:
IMO positive assertions are easier to read => `{isDesktop(layout) ? styles.desktopHeading : styles.tabletHeading}`